src/generator/common/helpers.ts

```diff
@@ -241,11 +240,20 @@ export function getEnums(schemas: Schemas): EnumEntry[] {
     enums.push(entry);
   }
   for (const choice of values(schemas.sealedChoices)) {
-    const entry = new EnumEntry(choice.language.go!.name, choice.choiceType.language.go!.name, choice.language.go!.possibleValuesFunc, choice.choices);
-    if (HasDescription(choice.language.go!)) {
-      entry.desc = choice.language.go!.description;
+    // TODO check what to do in the case of having a sealedChoice that doesnt have the choiceType field
```

Comment:
Since our enums are always strings we could hard-code the type and skip checking `choice.choiceType`.  Do you know what enum(s) in the blob storage swagger are missing this?

Reply:
There's a ContentType enum and it only has one choice.

Reply:
Removed it from the sealed-choice transform and helper func that would add it as an enum. They still show up on the model however.